soroban-env-common/src/symbol.rs

```diff
@@ -167,21 +166,27 @@ impl SymbolSmall {
 
 impl Symbol {
     #[doc(hidden)]
-    pub const unsafe fn from_small_body(body: u64) -> Self {
-        // This is a helper that should only be used in macros where it's
-        // necessary to generate a Symbol constant with no possibility of error:
-        // it just forces the bits that might be set wrong (54 and 55) to zero.
-        let body = body & SMALL_MASK;
-        Symbol(SymbolSmall::from_body(body).0)
```

Comment:
What's the motivation for removing the `from_small_body` function? Unclear to me if it's related to the change, or a drive-by change.